http.go

```diff
@@ -181,8 +174,8 @@ func (h *RasterHttpServer) handleImage(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Clean up the URL path into a local filename.
-	filename := sanitizeFilename(r.URL.Path)
-	storagePath := h.cache.GetFileName(r.URL.Path)
+	filename := r.URL.Path[1:]
```

Comment:
You might want to use `filename` instead of `r.URL.Path` in the log statement below.